docs/.vuepress/config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   title: "zkSync: secure, scalable crypto payments", // adding title gives us a header with search box
-  description: "zkSync is a fully trustless user-centric zkRollup protocol for scaling payments and smart contracts on Ethereum.",
+  description: "zkSync is a fully trustless user-centric zkrollup protocol for scaling payments and smart contracts on Ethereum.",
```

Comment:
zkrollup -> zkRollup

Sorry, there is really a mess with naming and it was here before you joined us, i.e. "zk rollup vs zk-rollup v vs ZK Rollup vs ZK rollup", etc... 

But generally if we decided to write zk and rollup together (as one word), it should be written either as "ZkRollup" or zkRollup. I am fine with both approaches (but at some point we need to agree on a standard!). 

So please change it to zkRollup as it was before